application/cms/models.py

```diff
@@ -863,10 +865,13 @@ class DimensionClassification(db.Model):
 # This encapsulates common fields and functionality for chart and table models
 class ChartAndTableMixin(object):
     id = db.Column(db.Integer, primary_key=True)
-    classification_id = db.Column("classification_id", db.Integer, nullable=False)
-    includes_parents = db.Column(db.Boolean, nullable=False)
-    includes_all = db.Column(db.Boolean, nullable=False)
-    includes_unknown = db.Column(db.Boolean, nullable=False)
+    classification_id = db.Column("classification_id", db.String(255), nullable=True)
+    includes_parents = db.Column(db.Boolean, nullable=True)
+    includes_all = db.Column(db.Boolean, nullable=True)
+    includes_unknown = db.Column(db.Boolean, nullable=True)
+
+    built_object = db.Column(JSON)
```

Comment:
Mmm, interesting name. 🤔 This feels like it could at least benefit from a comment saying what it is, if we don't go for something more specific? Some possible options: `highcharts_object`, `highcharts_data`, `visualisation_data`, `visualisation_object`, `processed_data`?

Reply:
Yeah it's only `highcharts` for charts, not tables, so was after something generic. `visualisation_object` could work.  I'll definitely add a comment.

Or do we think it's worth moving them out of the Mixin altogether?  I'm not against this, and then we could give much better names... like `chart_object` and `table_object`?

Reply:
I think it probably makes sense to move these out of the mixin, partly in anticipation of these deviating a bit more in future?